### ISSUE TITLE
Replace vQmod by OCmod

### ISSUE THREAD

**Issue by Ecno92** [SendCloud/SendCloud-OpenCart#1]
OCmod is the OpenCart replacement for vQmod. The advantage of OCmod is that it is installed by default in OpenCart 2.

It may be a good idea to start replacing vQmod by OCmod if this is possible for the changes that are required by the extension. 

Also read: 
http://stackoverflow.com/questions/26333058/ocmod-implementation-opencart-2-0
http://forum.opencart.com/viewtopic.php?f=34&t=129261

**Comment by nosite** [comment]
Any updates about this one? 
Want to go with SendCloud for OC2, don't what tot use Vqmod...

**Comment by Ecno92** [comment]
Currently there are no ongoing developments for this module. Although VQmod should work fine I can understand the preference for OCmod despite it has less functionality than VQmod. As far as I can see the transition to OCmod should be not that difficult. I'll check this week what we can do regarding to this issue in the near future. 

If you feel adventurous you can do it yourself and create a pull request before we get our hands dirty.

**Comment by belmer83** [comment]
whats the big deal about installing VQmod 2.0? 
if you do, just rename soundcloud.xml to soundcloud.vqmod.xlm and upload it with the build in extention installeren..

(This information is missing in the docs...)

**Comment by Ecno92** [comment]
I'm not very familiar with the internals of both vQmod and OCmod, but as far as I know there is no big winner in this discussion.

OCmod still has less possibilities than VQmod. It is also called the "poor mans vQmod. (http://stackoverflow.com/questions/26333058/ocmod-implementation-opencart-2-0, http://forum.opencart.com/viewtopic.php?f=34&t=129261)

However OCmod is installed by default and seems to be able to do everything we need. It may even improve in the future since it should be the default according to the OC project. Beside that the implications of supporting both are relatively low since the functionality of this module is still limited. Because of that I prefer to give the user the choice and that we make a decision for one of the concepts in a later stage once we know if OCmod is generaly accepted over vQmod. 

My collegue will check the changes in pull request #3  this week and we will update the docs if necessary.

**Comment by belmer83** [comment]
I tested the OCmod and it works fine.

**Comment by Ecno92** [comment]
Thanks for testing @belmer83. 
We've also tested and we also did not experience any problems.
So we will merge this PR and update the documentation and version number. 

Also thanks for your feedback regarding the panel. We are aware of both issues and we've registered them in our issue tracker to be solved.

For other SendCloud Panel related issues it's advised to send them to contact@sendcloud.nl because it's a separate project from this open source OpenCart module project. However it's okay to do suggestions which affect both the panel and the development of this module like in issue #4
